lib/core/versions/latest/models/InternalDocumentModel.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Defines the internal generic document data structure that is agnostic to any particular external schema.
+ */
+export default interface InternalDocumentModel {
+  didUniqueSuffix: string;
```

Comment:
Shouldn't this just be `uniqueID`/ `uniqueSuffix` or something not related to DID now?

Reply:
We are still registering "DIDs", so the unique portion is still technically the "didUniqueSuffix".